test/MinCostFlow/test_mcf.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import shutil
+from pathlib import Path
+from src.mincostflow import MinCostFlow
+from test.MEO.test_meo import OUT_FILE
+
+TEST_DIR = 'test/MinCostFlow/'
+OUT_FILE = TEST_DIR+'output/mincostflow-output.txt'
+
+class TestMinCostFlow:
+    def test_mincostflow_required(self):
```

Comment:
If we want to run multiple graphs, we can use syntax like this TPS test case https://github.com/gitter-lab/tps/blob/master/workflow/tests/test_workflow.py#L16

We would add a new argument `graph` and then provide a list of the graphs that are the hard-coded values that argument can take.